src/app/lib/config/services/tigrinya.js

```diff
@@ -58,19 +58,19 @@ const service = {
             'ኣብ መአለሺኻ ንዘሎ ደጊምካ መፈተኒ መልጎም ጠውቕ',
             'ነቲ ናይ ቢቢሲ መአለሺ ስፍራ ተጠቒምካ ነዚ ገጽ ክትረኽቦ ፈትን',
           ],
-          callToActionFirst: 'ከም ኣማራጺ ከኣ ',
-          callToActionLinkText: '[BBC News ረኣይ።]',
-          callToActionLast: '',
+          callToActionFirst: 'ከም ኣማራጺ ከኣ   ',
```

Comment:
```suggestion
          callToActionFirst: 'ከም ኣማራጺ ከኣ ',
```

This seems to have 3 spaces (`...`) rather then a single space (`.`) - is this on purpose?

Reply:
not on purpose , just removed the`homepage` word and kept spaces by mistake